### PR TITLE
Add template files to show how to implement a new manifold in geomstats

### PR DESCRIPTION
**Pull request by ninamiolane**
This PR adds:
- one _my_manifold.py file, that shows how to implement a new manifold in geomstats,
- one test__my_manifold.py, that shows how to write the associated tests.

### CODE REVIEW

**Comment by ninamiolane** [comment]
I will clean so that Travis and Deepsource pass.

File: geomstats/geometry/_my_manifold.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    dim : int
+        Dimension of the manifold.
+    """
+
+    def __init__(self, dim, another_parameter, **kwargs):
+        super(MyManifold, self).__init__(dim)
+        self.another_parameter = another_parameter
+
+    # Implement the main methods of MyManifold, for example belongs:
+    def belongs(self, point):
```
Comment (nguigs) [inline geomstats/geometry/_my_manifold.py:42]: Mention that the signature of the method should match - when possible- the signature of the overriden method in the parent class. Here `belongs(self, point, atol=ATOL)`.

**Review by nguigs — APPROVED** [review]
Very nice! It will definitely help new contributors :)

File: geomstats/geometry/_my_manifold.py (hunk 1 above)
Comment (ninamiolane) [reply geomstats/geometry/_my_manifold.py:42]: Good point, done!

File: geomstats/geometry/_my_manifold.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        """Give a one-liner description of the method.
+
+        For example: Evaluate if a point belongs to MyManifold.
+
+        List the parameters of the method.
+        For example:
+
+        Parameters
+        ----------
+        point : array-like, shape=[..., dim]
```
[the remark this answers is not in the document]
Comment (ninamiolane) [reply geomstats/geometry/_my_manifold.py:52]: Yes, done.

**Comment by ninamiolane** [comment]
@tristancabel here is the PR that shows how to implement a new manifold in geomstats - do you want to do the code review?

**Pull request merged**